src/NuGetPackageVerifier/Rules/AuthenticodeSigningRule.cs

```diff
@@ -17,14 +17,10 @@ public class AuthenticodeSigningRule : IPackageVerifierRule
             IPackageMetadata package,
             IPackageVerifierLogger logger)
         {
-            var packagePath = nupkgFile.FullName;
-            var nupkgWithoutExt = Path.Combine(
-                Path.GetDirectoryName(packagePath),
-                Path.GetFileNameWithoutExtension(packagePath));
-
+            var extractPath = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName());
```

Comment:
I'll move this into a separate commit. This lets us run the verification on directories without write permissions (such as the CI drop share)